### PR TITLE
Adds a disposal chute to the command meeting room on delta station

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
You can now clean up trash in the meeting room and or the bridge so it looks cleaner

## Why It's Good For The Game
Bridge not soo messy covered in trash on delta station

## Images of changes
 
Screenshot 2024-06-23 095044

## Testing
Spawned as a the captain and thru a paper away and made sure it ended up in cargo on delta station

## Changelog
:cl:
fix: Adds a disposal chute to the command meeting room so you can now throw your trash away on delta station
/:cl:

### CODE REVIEW

**Review by Sirryan2002 — APPROVED** [review]

**Review by S34NW — DISMISSED** [review]
Please use the corner subtype, not the var edited segment.

image
image

**Comment by Fluffyzack21** [comment]
> Please use the corner subtype, not the var edited segment.
> 
> image image

I am looking over it and it saved as that, I triple checked and then I tried to save it nothing comes up that got changed

File: _maps/map_files/stations/deltastation.dmm [hunk 1]
```diff
@@ -27970,6 +27966,10 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
+/obj/structure/disposalpipe/segment{
+	dir = 4;
+	icon_state = "pipe-c"
+	},
```
Comment (S34NW) [inline _maps/map_files/stations/deltastation.dmm:27972]: You've still got a var edited corner pipe

File: _maps/map_files/stations/deltastation.dmm [hunk 2]
```diff
@@ -84060,6 +84131,20 @@
...
 /area/station/hallway/primary/aft/north)
+"ptz" = (
+/obj/machinery/atmospherics/pipe/simple/hidden/supply,
+/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/structure/cable{
+	d1 = 1;
+	d2 = 2;
+	icon_state = "1-2"
+	},
+/obj/structure/disposalpipe/segment{
```
Comment (S34NW) [inline _maps/map_files/stations/deltastation.dmm:84142]: and here

Comment (Fluffyzack21) [reply _maps/map_files/stations/deltastation.dmm]: fixed

File: _maps/map_files/stations/deltastation.dmm (hunk 1 above)
Comment (Fluffyzack21) [reply _maps/map_files/stations/deltastation.dmm]: fixed

**Review by warriorstar-orion — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**